packages/apps/correios-v2/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "@cloudcommerce/app-correios-v2",
+  "type": "module",
+  "version": "0.0.0",
+  "description": "E-Com Plus Cloud Commerce app to integrate Correios new APIs",
+  "main": "lib/index.js",
+  "exports": {
+    ".": "./lib/index.js",
+    "./events": "./lib/correios-events.js"
```

> Sobre https://github.com/ecomplus/cloud-commerce/pull/251#discussion_r1370378728 e https://github.com/ecomplus/cloud-commerce/pull/251#discussion_r1370395472
Esse package.json novo não deveria existir, mas no do `app-correios`:

```suggestion
    "./events": "./lib/correios-events.js",
    "./fill-database": "./lib-mjs/fill-database.mjs"
```